namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-equinity-historical-data-prod/resources/s3.tf

```diff
@@ -109,44 +109,47 @@ module "s3_bucket" {
       ]
     },
   ]
-
   */
 
-  /*
-   * The following are exampls of bucket and user policies. They are treated as
-   * templates. Currently, the only available variable is `$${bucket_arn}`.
-   *
-   */
-
-  /*
- * Allow a user (foobar) from another account (012345678901) to get objects from
- * this bucket.
- *
-
-   bucket_policy = <<EOF
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Principal": {
-        "AWS": "arn:aws:iam::012345678901:user/foobar"
+  bucket_policy = <<EOF
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Sid": "AllowBucketAccess",
+        "Effect": "Allow",
+        "Principal": {
+          "AWS": "${module.s3_bucket.bucket_arn}"
+        },
+        "Action": [
+          "s3:ListBucket",
+          "s3:GetBucketLocation",
+        ],
+        "Resource": [
+          "${module.s3_bucket.bucket_arn}",
```

Comment:
add another resource like this `"${module.s3_bucket.bucket_arn}/*"`

Reply:
addressed on refactored version